homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -218,6 +229,16 @@ async def _initialize(self) -> None:
 
         await self._async_update_sound_modes()
 
+    async def async_update(self) -> None:
+        """Update queue settings."""
+        # The WebSocket event listener is the main handler for connection state.
+        # The polling updates do therefore not set the device as available or unavailable
+        with contextlib.suppress(ApiException, ClientConnectorError, TimeoutError):
```

Comment:
Should we rather catch them and log something?

Reply:
The WebSocket listener already logs when connection is lost, so if this also logs there would be two log messages informing the user of pretty much the same thing.